frontend/server/src/main/java/com/amazonaws/ml/mms/util/messages/WorkerCommands.java

```diff
@@ -10,7 +10,9 @@
     @SerializedName("unload")
     UNLOAD("unload"),
     @SerializedName("stats")
-    STATS("stats");
+    STATS("stats"),
+    @SerializedName("stop")
```

Comment:
I don't think we need stop command.

Reply:
I added this to make sure that the thread is stopped before restarting. This is basically the poison pill method.